wallet/src/de/schildbach/wallet/service/BlockchainServiceImpl.java

```diff
@@ -461,6 +461,9 @@ public void onPreBlockProgressUpdated(PreBlockStage stage) {
             if (stage == PreBlockStage.Starting && lastPreBlockStage == PreBlockStage.None) {
                 startPreBlockPercent = syncPercentage;
             }
+            if (preBlocksWeight > 0.99) {
```

Comment:
Is this related to the ticket?

Reply:
This is not related to the ticket, but does address a cause of having a sync status > 100%.

Reply:
This change was moved to https://github.com/dashevo/dash-wallet/pull/608